### ISSUE TITLE
TypeError: Cannot read property 'Prefetch' of undefined

### ISSUE THREAD

**Issue by stereodenis** [asgerjensen/typedoc-plugin-external-module-map#8]
```
typedoc src
Loaded plugin /Users/stereodenis/work/busfor-app/node_modules/typedoc-plugin-external-module-map

Using TypeScript 3.7.5 from /Users/stereodenis/work/busfor-app/node_modules/typedoc/node_modules/typescript/lib
/Users/stereodenis/work/busfor-app/node_modules/typedoc-plugin-external-module-map/plugin.js:46
            this.options.read({}, options_1.OptionsReadMode.Prefetch);
                                                            ^

TypeError: Cannot read property 'Prefetch' of undefined
    at ExternalModuleMapPlugin.onBegin (/Users/stereodenis/work/busfor-app/node_modules/typedoc-plugin-external-module-map/plugin.js:46:61)
    at triggerEvents (/Users/stereodenis/work/busfor-app/node_modules/typedoc/dist/lib/utils/events.js:128:43)
    at triggerApi (/Users/stereodenis/work/busfor-app/node_modules/typedoc/dist/lib/utils/events.js:110:13)
    at eventsApi (/Users/stereodenis/work/busfor-app/node_modules/typedoc/dist/lib/utils/events.js:21:18)
    at Converter.trigger (/Users/stereodenis/work/busfor-app/node_modules/typedoc/dist/lib/utils/events.js:264:13)
    at Converter.convert (/Users/stereodenis/work/busfor-app/node_modules/typedoc/dist/lib/converter/converter.js:89:14)
    at CliApplication.convert (/Users/stereodenis/work/busfor-app/node_modules/typedoc/dist/lib/application.js:72:39)
    at CliApplication.bootstrap (/Users/stereodenis/work/busfor-app/node_modules/typedoc/dist/lib/cli.js:45:34)
    at Object.<anonymous> (/Users/stereodenis/work/busfor-app/node_modules/typedoc/bin/typedoc:5:5)
```

**Comment by maneetgoyal** [comment]
- Ran into the same bug.
- Using TypeDoc version: `"^0.16.11"`
- The [breaking change in `v0.16.0/0.16.1`](https://github.com/TypeStrong/typedoc/releases/tag/v0.16.0) might be the reason behind this.

**Comment by danbalarin** [comment]
Thanks to @maneetgoyal it was easy to track down error. I fixed it in PR #9 ,hopefully will be merged soon

**Comment by maneetgoyal** [comment]
Hi @asgerjensen, a fix would be really helpful. Currently stuck with older typescript version. Please let us know if there’s anything on our end we can do. Many thanks for maintaining this repo though :)

**Comment by asgerjensen** [comment]
Hi,

Thanks for the PR. 

Its merged and built as 1.1.0

I also bumped all deps and built as 1.2.1. 

Sendt fra min iPhone
